### PR TITLE
link to julia_pod in README

### PR DESCRIPTION
**Pull request by kolia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 A Julia cluster manager for provisioning workers in a Kubernetes (K8s) cluster.
 
+Pairs well with [julia_pod](https://github.com/beacon-biosignals/julia_pod) for k8s-native julia development.
```
Comment (ararslan) [inline README.md:10]:
```suggestion
Pairs well with [julia_pod](https://github.com/beacon-biosignals/julia_pod) for K8s-native Julia development.
```
Just makes capitalization consistent with the sentence above it.

**Review by ararslan — APPROVED** [review]
Feel free to merge once the above is applied

File: README.md (hunk 1 above)
Comment (omus) [inline README.md:10]:
```suggestion
Pairs well with [`julia_pod`](https://github.com/beacon-biosignals/julia_pod) for interactive Julia development within a K8s pod. 
```

Comment (ararslan) [reply README.md:10]: This one is better than mine

**Pull request merged**